_projects/2022/100385774/100385774.Rmd

```diff
@@ -0,0 +1,902 @@
+
+knitr::include_graphics("Data/Original plot.jpg")
+
+```
+
+
+With it, their objective was to show how despite all controversies and negative feelings towards them, a majority of Northamericans still use at least one, and how their usage has increased through the last years. The goal of this project is to replicate the plot as closely as posible and to offer feasible enhancements and alternative visualizations. 
+
+## Plot construction
+
+### Initial settings, getting the data
+
+First of all, we should load into R the necessary libraries to develop our work. In our case, we need the following three: 
+
+```{r}
+
+library(tidyverse) #To use the pipe operator and filter data
+library(ggplot2) #To produce the graph
+library(showtext) #To include alternative fonts
+
+```
+
+Once we prepared our environment, it's time for us to work with the data. In this case, the Pew Research Center uploaded a PDF file (available in Addendum 2) with the specific questions and answers used for the report, from which we created a CSV file. We anticipated our future needs and introduced the different variables in a long format, as we would later need it to develop the plot. 
+
+```{r}
+
+read_delim("Data/database.csv")
+
+```
+
+So, we store our CSV file as a dataframe, also renaming the 'Social media' variable into just 'Social' for simplicity purposes. Additionally, we filter the first and last data values for each Social Network into another dataframe, as we later will need it to display the points in each side of the lines. 
+
+```{r}
+
+data <- read_delim("Data/database.csv")
+
+data <- data %>% rename(
+  "Social" = `Social media`
+)
+
+filtered <- data %>% 
+  group_by(Social) %>% 
+  filter(Year==min(Year)|Year==max(Year))
+
+```
+
+### Font preparation and color setting
+
+In this step, we start by adding into our document the fonts used by the authors. Note that we included the same font two times, the regular and the bold version; this is because we had some issues while trying to compute the bold aspect manually (with double asterisk at each side of the text).
+
+```{r}
+
+font_add(family = "Franklin Gothic", regular = "FRAMD.ttf")
+font_add(family = "Franklin Gothic Demi", regular = "FRADM.ttf")
+font_add(family = "Georgia", regular = "GEORGIAI.ttf")
+showtext_auto()
+
+```
+
+Then, we create a vector assigning each possible Social Network to its displayed color in the graph, to then apply it to the lines setting. 
+
+```{r}
+
+colors <- c("Youtube" = "#bf4538", "Facebook" = "#23689a", 
+            "Instagram" = "#6c9acf", "Pinterest" = "#747a46",
+            "Linkedin" = "#c2c88e", "Snapchat" = "#746a7c", 
+            "Twitter" = "#d1aa42", "Whatsapp" = "#253a5c",
+            "TikTok" = "#e9a03f", "Reddit" = "#92885a", 
+            "Nextdoor" = "#e28377" )
+
+```
+
+
+### Coordinates, axes and scales
+
+Our panel base depends on two axis, years and usage rate (%). 
+
+```{r, fig.height=6, fig.width=6, out.height="100%", out.width="100%", dpi=300}
```

Comment:
This DPI is too much. The images are too big with this. Please remove this option and leave the default. Is there any issue with the default?

Reply:
(The same for all the other chunks)